src/LLVM_Runtime_Linker.cpp

```diff
@@ -941,6 +973,9 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
             if (t.arch == Target::POWERPC) {
                 modules.push_back(get_initmod_powerpc_ll(c));
             }
+            if (t.arch == Target::RISCV) {
+              //                modules.push_back(get_initmod_riscv_ll(c));
```

Comment:
Some explanation for why this is commented out but still exists would be nice. Is it because this is where an .ll runtime module will go once we have one?